.github/workflows/prerelease.yml

```diff
@@ -85,9 +85,8 @@ jobs:
           flavor: |
             latest=false
           tags: |
-            type=semver,pattern={{version}}+py${{ matrix.python-version }}
+            type=match,pattern=v(.*),group=1,suffix=-py${{ matrix.python-version }}
             type=semver,pattern={{version}},enable=${{ matrix.python-version == 3.8 }}
-            type=raw,value={{version}}-py${{ matrix.python-version }}r
```

Comment:
This caused the previous run to fail since there are no template arguments for `type=raw`
- Previous run: https://github.com/nautobot/nautobot/actions/runs/5193439184/jobs/9365530741#step:8:50
- Docs: https://github.com/docker/metadata-action#typeraw